playbooks/ansible/roles/sit.gpfs/tasks/new_share.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Bind the GPFS volume to mount path
+  mount:
+    path: "{{ config.paths.mount }}/{{ volume }}"
+    src: "/gpfs/sit_fs/{{ volume }}"
+    opts: bind
+    fstype: none
+    state: mounted
```

Comment:
IMO above bind mount task is better fit for _new_volume.yml_.

Reply:
I agree. To not make this too hacky in the current approach, I'll also make a change where the volume for the shares is created from _new_share.yml_ instead of the backend installation. This also seems more logical to me.

Reply:
Check my previous comment. Bind needs to be done on all server nodes, but _new_volume.yml_ is only called once.